node_modules/col-course/lib/api.js

```diff
@@ -55,7 +55,7 @@ var getOrCreateCourse = module.exports.getOrCreateCourse = function(canvasCourse
   DB.Course.findOrCreate(options).complete(function(err, data) {
     if (err) {
       log.error({'err': err}, 'Failed to get or create a course');
-      return callback(err);
+      return callback({'code': 500, 'msg': err.message});
```

Comment:
@nicolaasmatthijs : just out of curiosity, is status code + message a convention for error objects in Node, or do people build them differently in different contexts?